lib/dor/text_extraction/ocr.rb

```diff
@@ -20,20 +21,29 @@ def abbyy_input_path
         File.join(Settings.sdr.abbyy.local_ticket_path, bare_druid)
       end
 
+      # rubocop:disable Metrics/MethodLength
       # rubocop:disable Metrics/AbcSize
       def cleanup
         if Dir.exist?(abbyy_input_path)
           raise "#{abbyy_input_path} is not empty" unless Dir.empty?(abbyy_input_path)
 
+          # e.g. /abbyy/INPUT/ab123cd4567
+          logger.info "Removing empty ABBYY input directory: #{abbyy_input_path}"
           FileUtils.rm_rf(abbyy_input_path)
         end
 
-        return true unless Dir.exist?(abbyy_output_path)
-
```

Comment:
do not need this line since `FileUtils.rm_rf` doesn't fail if the folder isn't there